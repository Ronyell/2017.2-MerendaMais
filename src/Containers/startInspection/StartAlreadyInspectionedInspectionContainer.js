import { connect } from 'react-redux';
import StartAlreadyInspectionedInspection from '../../screens/startInspection/StartAlreadyInspectionedInspection';
import { resetStore } from '../../actions/applicationActions';
import { asyncGetCurrentPost } from '../../actions/reportResultActions';

const mapStateToProps = state => (
  {
    isLoading: state.application.isLoading,
    counselor: state.counselor,
    listOfAlreadyInpectionedSchedueInAGroup:
      state.list.listOfAlreadyInpectionedSchedueInAGroup,
  }
);

const mapDispatchToProps = dispatch => ({
  asyncGetCurrentPost: getData => dispatch(asyncGetCurrentPost(getData)),
  resetStore: (...states) => dispatch(resetStore(...states)),
});

const StartAlreadyInspectionedInspectionContainer = connect(mapStateToProps,
  mapDispatchToProps)(StartAlreadyInspectionedInspection);

export default StartAlreadyInspectionedInspectionContainer;
