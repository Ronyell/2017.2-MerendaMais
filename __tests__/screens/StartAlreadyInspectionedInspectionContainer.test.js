import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import configureStore from 'redux-mock-store';
// import configureStore from 'redux-mock-store';
import StartAlreadyInspectionedInspectionContainer from '../../src/Containers/startInspection/StartAlreadyInspectionedInspectionContainer';

Enzyme.configure({ adapter: new Adapter() });

const mockStore = configureStore();

const initialState = {

  application: {
    isLoading: false,
  },

  list: {
    listOfExpiredScheduleInAGroup: {
      codSchool: '',
      date: '',
      time: '',
    },
  },

  reportResult: {
    isLoadingResult: false,
  },
};

const store = mockStore(initialState);

describe('Testing StartPendingInspection Screen', () => {
  it('renders as expected', () => {
    const wrapper = shallow(
      <StartAlreadyInspectionedInspectionContainer />,
      { context: { store } },
    );
    expect(wrapper).toMatchSnapshot();
  });
});
