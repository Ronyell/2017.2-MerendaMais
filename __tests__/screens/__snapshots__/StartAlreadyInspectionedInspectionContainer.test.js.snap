// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing StartPendingInspection Screen renders as expected 1`] = `
<StartAlreadyInspectionedInspection
  asyncGetCurrentPost={[Function]}
  isLoading={false}
  resetStore={[Function]}
/>
`;
